### ISSUE TITLE
Comment with club

### ISSUE THREAD

**Issue by sachabinder** [KIClubinfo/upont-v4#100]
Cannot comment with club on a post.

**Comment by LudoBike** [comment]
I looked at it bug I fail to correct it 

For me the issue is in the [view posts in news framework](https://github.com/KIClubinfo/upont-v4/blob/60879529691672f18e80674d18521c8b5e12c7f6/back/news/views.py#L18) but I'm not sure.

I can also come from the connection between the front and the back. I noticed that if the user post a comment without acting with the author selection list, the front sent an empty field for club and the comment is posted with his/her user name but if he/she selects "Élèves" the front sent "-1" for the club field and no comment is posted.

I've tried to rewrite the view and the form [CommentForm](https://github.com/KIClubinfo/upont-v4/blob/60879529691672f18e80674d18521c8b5e12c7f6/back/news/forms.py#L78) unsuccessfully.
